sw/airborne/modules/computer_vision/snake_gate_detection.c

```diff
@@ -46,14 +46,14 @@
 #include "paparazzi.h"
 
 // to debug the algorithm, uncomment the define:
-// #define DEBUG_SNAKE_GATE
+#define DEBUG_SNAKE_GATE
```

Comment:
do you really want to keep this enabled by default ? all the other ones are disabled

Reply:
Good catch! Changed it.